tests/tests/test_basic_integration.py

```diff
@@ -156,22 +125,13 @@ def do_test_failed_updated_and_valid_update(
         )
 
     def do_test_update_no_compression(self, env, valid_image_with_mender_conf):
-        """Uploads an uncompressed artifact, and runs the whole update process."""
+        """Uploads an uncompressed artifact, and runs the whole udpate process."""
```

Comment:
This specific line would be nice to drop from the revert or re-aply in a new commit.

Reply:
I'll re-apply it.